test/suites/smoke-test-dancelight/test-era-changes.ts

```diff
@@ -0,0 +1,119 @@
+import "@tanssi/api-augment/dancelight";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import type { ApiPromise } from "@polkadot/api";
+import { getCurrentEraStartBlock, getPastEraStartBlock } from "utils/block";
+
+describeSuite({
+    id: "SMOKD02",
+    title: "Era changes suit that only runs on Dancelight chains",
+    foundationMethods: "read_only",
+    testCases: ({ it, context, log }) => {
+        let api: ApiPromise;
+        let currentEraIndex: number;
+        let currentEraStartBlock: number;
+        let pastEraStartBlock: number;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+            currentEraStartBlock = await getCurrentEraStartBlock(api);
+            currentEraIndex = (await api.query.externalValidators.activeEra()).unwrap().index.toNumber();
+            pastEraStartBlock = await getPastEraStartBlock(api, currentEraStartBlock - 1);
+        });
+
+        it({
+            id: "C01",
+            title: "Era changes are happening as expected and information is correct",
+            test: async () => {
+                const sessionsPerEra = api.consts.externalValidators.sessionsPerEra.toNumber();
+                const apiAtCurrentEraStart = await api.at(await api.rpc.chain.getBlockHash(currentEraStartBlock));
+                const currentEraStartSessionIndex = (
+                    await apiAtCurrentEraStart.query.session.currentIndex()
+                ).toNumber();
+                const apiAtPreviousEraStart = await api.at(await api.rpc.chain.getBlockHash(pastEraStartBlock));
+                const previousEraStartSessionIndex = (
+                    await apiAtPreviousEraStart.query.session.currentIndex()
+                ).toNumber();
+
+                expect(previousEraStartSessionIndex + sessionsPerEra).toEqual(currentEraStartSessionIndex);
```

Comment:
Can we add a text message with params, also let's add the block number we are checking, so it is easier to debug, in case of fail.

Reply:
Yup added error messages and more checks for the era change smoke test in https://github.com/moondance-labs/tanssi/pull/1119/commits/7197245b5980021fc7058ec962207c29661ade4f